### PR TITLE
Feature/contentful metadata

### PR DESCRIPTION
**Pull request by weberjavi**
This PR adds contentful metadata to the store to fill the metadata modal of the different layers on the sidebar.
Kapture 2019-06-10 at 14 55 59

### CODE REVIEW

File: src/components/radio-group/radio-group-component.jsx [hunk 1]
```diff
@@ -1,39 +1,23 @@
...
-  options.forEach(option => {
-    Object.keys(option.layers).forEach(variant => {
-      optionsLayers.push({
-        layerTitle: option.layers[variant],
-        variant,
-        option
-      });
-    });
-  });
+  const [variant, setVariant] = useState(RARITY);
```
Comment (tsubik) [inline src/components/radio-group/radio-group-component.jsx]: Hmm, that won't work. As the selected variant source of truth is the active layer that info should be taken from activeLayers. When this component is destroyed we will use that inner state. Try to close Biodiversity Layers and open them again. You will see that it is not selecting radio buttons correctly.

**Comment by weberjavi** [comment]
@tsubik ☝️ ready again

File: src/components/checkbox-group/checkbox/checkbox.js [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
 import Component from './checkbox-component';
+import actions from 'redux_modules/metadata';
 
-export default Component;
\ No newline at end of file
+const CheckboxContainer = props => {
+
+  const handleInfoClick = (option) => {
+    const { setModalMetadata } = props;
+    console.log(option)
```
Comment (tsubik) [inline src/components/checkbox-group/checkbox/checkbox.js]: forgotten console log

Comment (weberjavi) [reply src/components/checkbox-group/checkbox/checkbox.js]: -.-

**Comment by weberjavi** [comment]
ok, already removed.
so... merging

**Pull request merged**